tap_fullstory/__init__.py

```diff
@@ -81,11 +81,10 @@ def on_giveup(details):
 
 @backoff.on_exception(backoff.expo,
                       requests.exceptions.RequestException,
-                      max_tries=5,
+                      max_tries=6,
```

Comment:
After the initial burst allowance has been exceeded, the rate limit is 2/minute. This additional retry makes sure that we wait long enough for the next token to become available.